### PR TITLE
Add Xenial builds to the CTest driver script

### PR DESCRIPTION
**Pull request by BetsyMcPhail**

### CODE REVIEW

**Comment by BetsyMcPhail** [comment]
@jamiesnape  and @mwoehlke-kitware  PTAL. I am not sure if the set-up script is being called in the best way... Suggestions/comments please.


One other issue, drake requires CMake 3.5 while the ctest_driver_script requires CMake 3.6. I think the solution is to install CMake 3.6 on the xenial AMI. Then the install script can check if CMake is already installed before installing the default version (3.5). And, of course, CMake needs to be installed on the image before it can run this script!

**Comment by jamiesnape** [comment]
Yes, though you will need to install CMake 3.7 on the AMI once it is out. While we are still testing clone the latest release branch, build from source, and install to `/usr/local/bin` (i.e., the default install location).

File: ctest_driver_script.cmake [hunk 1]
```diff
@@ -251,6 +262,10 @@ if(COMPILER MATCHES "^scan-build")
   file(MAKE_DIRECTORY "${DASHBOARD_CCC_ANALYZER_HTML}")
 endif()
 
+if(COMPILER MATCHES "^xenial")
+  execute_process(COMMAND bash "-c" "yes | sudo ${DASHBOARD_WORKSPACE}/setup/ubuntu/16.04/install_prereqs.sh")
+endif()
```
Comment (jamiesnape) [inline ctest_driver_script.cmake]: You need to check the return value and fail if the script fails.

**Comment by BetsyMcPhail** [comment]
PR for set-up script changes: [#3806](https://github.com/RobotLocomotion/drake/pull/3806)

The latest release branch of CMake is installed on the Xenial AMI from today (2016-10-24).

**Comment by jamiesnape** [comment]
Can you add support for an environment variable `provision` that is set to `true` or `false`?

**Comment by jamiesnape** [comment]
Do the values of the `CC`, `CXX`, `FC`, `F77` environment variables need to be changes for Xenial?

File: ctest_driver_script.cmake [hunk 2]
```diff
@@ -251,6 +264,21 @@ if(COMPILER MATCHES "^scan-build")
   file(MAKE_DIRECTORY "${DASHBOARD_CCC_ANALYZER_HTML}")
 endif()
 
+if (PROVISION)
```
Comment (jamiesnape) [inline ctest_driver_script.cmake]: `if(`

**Pull request merged**